src/views/Characters/Characters.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useState, useEffect } from 'react';
+import { fetchCharacters } from '../../services/characters';
+import CharactersList from '../../components/Characters/CharactersList';
+import Controls from '../../components/Controls/Controls';
+
+export default function Characters() {
+  const [character, setCharacter] = useState([]);
+  const [query, setQuery] = useState('');
+  const [race, setRace] = useState('All');
+
+  useEffect(() => {
+    const getCharacter = async () => {
+      const response = await fetchCharacters(race, query);
```

Comment:
No need to pass query in is an argument to the fetchCharacters function inside the useEffect. The fetchCharacter function has it's second parameter set to a default value.